proto/regen/ecocredit/v1/tx.proto

```diff
@@ -165,6 +190,34 @@ message MsgCreateBatchResponse {
   string batch_denom = 1;
 }
 
+// MsgMintBatchCredits is a request type for MintBatchCredits Msg RPC.
+message MsgMintBatchCredits {
+  // Issuer must equal to the batch.issuer address.
+  // Signer of the msg.
+  string issuer = 1;
+
+  // batch_denom is the unique ID of the credit batch.
+  string batch_denom = 2;
+
+  // issuance are the credits issued in the batch.
+  repeated BatchIssuance issuance = 3;
+
+  // open=true keeps the batch unlocked for future minting.
+  // Othrwise we will seal the batch after the mint.
+  // Throws an error if the batch is already sealed.
+  bool open = 4;
```

Comment:
Could we update this to actually be a separate message? I think in most use cases (thinking of bridges), the issuer won't actually ever want to seal the batch.  Adding this here requires the user to explicitly set this value every time they call `MsgMintBatchCredits`, and if they forget, then it would lock the batch and prevent the issuer from being able to issue more credits to that vintage (assuming we have the appropriate restrictions on project <> credit type duplicate issuances).

I think I'd rather see something like:
```proto
message MsgSealBatch {
  string issuer = 1;
}
```

Reply:
@aaronc I know you had initially proposed this `bool open = 4;` what do you think of this upgrade?

Reply:
I don't think it makes that big of a difference. This should all be done automatically anyway. But I don't object

Reply:
Yesterday we had a session with Cory about the process and we found that it would make sense to separate the toggle functionality to a separate message. So if nobody objects I will make the update.